pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu

```diff
@@ -172,6 +173,19 @@ RUN apt-get update \
  && mv /etc/ssh/ssh_config.new /etc/ssh/ssh_config \
  && rm -rf /var/lib/apt/lists/*
 
+# Configure OpenSSH so that nodes can communicate with each other
+RUN mkdir -p /var/run/sshd && \
+  sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+
+RUN rm -rf /root/.ssh/ && \
+ mkdir -p /root/.ssh/ && \
+ ssh-keygen -q -t rsa -N '' -f /root/.ssh/id_rsa && \
+ cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n StrictHostKeyChecking no\n" >> /root/.ssh/config
+
+# Comment out line in MPI config to prevent mutually exclusive MCA settings
+RUN sed -i '62,62 s/^/#/' /home/.openmpi/etc/openmpi-mca-params.conf
```

Comment:
Is there a cleaner way to do this?

Reply:
to just remove the lines that add those openmpi mca params?

Reply:
Yeah